src/app/api/Api.ts

```diff
@@ -274,9 +281,10 @@ export interface API {
 
     // Sheriff Shifts
     getShifts(): Promise<Shift[]>;
-    updateShift(shiftToUpdate: Partial<Shift>): Promise<Shift>;
+    updateSelectedShifts(shiftIds: IdType[], shiftUpdates: ShiftUpdates): Promise<Shift[]>;
```

Comment:
here I think we could change the name of this method to be more generic (i.e. `updateManyShifts`) since the fact that the shifts are selected doesn't really matter to this call, that's more of the concern of the UI component that happens to be using this method.